src/Surfnet/StepupGateway/U2fVerificationBundle/Service/RegistrationVerificationResult.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+/**
+ * Copyright 2015 SURFnet bv
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\StepupGateway\U2fVerificationBundle\Service;
+
+use Surfnet\StepupGateway\U2fVerificationBundle\Entity\Registration;
+use Surfnet\StepupGateway\U2fVerificationBundle\Exception\LogicException;
+use Surfnet\StepupGateway\U2fVerificationBundle\Value\KeyHandle;
+use Surfnet\StepupGateway\U2fVerificationBundle\Value\PublicKey;
+use Surfnet\StepupU2fBundle\Service\RegistrationVerificationResult as U2fRegistrationVerificationResult;
+
+/**
+ * @SuppressWarnings(PHPMD.TooManyMethods)
+ */
+final class RegistrationVerificationResult
+{
+    /**
+     * @var U2fRegistrationVerificationResult
+     */
+    private $wrappedResult;
```

Comment:
if it looks like a duck, acts like a ducks and sounds like a duck then call it a duck...